### PR TITLE
keep make targets in order.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@JohnStrunk PTAL :)

File: Makefile [hunk 1]
```diff
@@ -14,6 +10,19 @@ check-go:
 	@./scripts/check-go.sh
 	@echo
 
+check-reqs:
+	@./scripts/check-reqs.sh
+	@echo
+
```
Comment (Madhu-1) [inline Makefile]: can you remove one extra line here?

Comment (humblec) [reply Makefile]: sure

Comment (humblec) [reply Makefile]: @Madhu-1 but I dont see it

Comment (Madhu-1) [reply Makefile]: @humblecI see 2 empty lines [line 16 and line 17]

Comment (humblec) [reply Makefile]: ah. yes, done. Thanks

File: Makefile [hunk 2]
```diff
@@ -14,6 +10,18 @@ check-go:
...
+glusterfs-csi-driver:
+	@echo Building glusterfs csi driver
+	go build -o build/glusterfs-csi-driver  cmd/glusterfs/main.go
+
```
Comment (Madhu-1) [inline Makefile]: @humblec can we add an `@echo` after `go build`

Comment (humblec) [reply Makefile]: sure

**Pull request merged**